pkg/controller/common/util.go

```diff
@@ -2139,7 +2142,11 @@ func AllowClaimAdoption(c client.Client, pvc *corev1.PersistentVolumeClaim, dv *
 	if pvc == nil || dv == nil {
 		return false, nil
 	}
-	anno, ok := dv.Annotations[AnnAllowClaimAdoption]
+	anno, ok := pvc.Annotations[AnnCreatedForDataVolume]
+	if ok && anno == string(dv.UID) {
```

Comment:
Maybe I misunderstood the annotation but shouldn't we allow adoption when the annotation matches the dv UID?

Reply:
No if UID matches we want to do regular population.  During DR restore the DV will have a different UID

Reply:
Aw right, that makes total sense. Thanks.